packages/calling-stateful-client/src/CallSubscriber.ts

```diff
@@ -168,6 +161,15 @@ export class CallSubscriber {
 
   private stateChanged = (): void => {
     this._context.setCallState(this._callIdRef.callId, this._call.state);
+
+    // subscribe to captions here so that we don't call captions when call is not initialized
+    if (_isTeamsMeetingCall(this._call) && this._call.state === 'Connected') {
```

Comment:
- is this the only event we can subscribe to after we connect to a call?
- what if we were in two calls and want to have captions in both as we jump between the calls?
- could we potentially expose a new "event" to subscribe to calls and this function only gets fired once the first time we connect to the main call?